File: sdk/java-sdk/src/main/java/com/evernym/verity/sdk/exceptions/UndefinedContextException.java
package com.evernym.verity.sdk.exceptions;

/**
 * Exception class used when required data is not contained in given Context object
 */
public class UndefinedContextException extends VerityException {
    /**
     * Constructs a new UndefinedContextException with the given message.
     *
     * @param message the given message for the cause of the exception.
     *
     * @see VerityException#VerityException(String)
     */
    public UndefinedContextException(String message) {
        super(message);
    }
}

File: sdk/dotnet-sdk/src/VeritySDK/Exception/UndefinedContextException.cs
using System;

namespace VeritySDK
{
    /// <summary>
    /// Exception class used when required data is not contained in given Context object
    /// </summary>
    public class UndefinedContextException : VerityException
    {
        /// <summary>
        /// Constructs a new exception with the given message.
        /// </summary>
        /// <param name="message">message the given message for the cause of the exception.</param>
        public UndefinedContextException(String message) : base(message) { }
    }
}
